massa-execution-exports/src/event_store.rs

```diff
@@ -87,6 +87,11 @@ impl EventStore {
                     (Some(_), None) => return false,
                     _ => (),
                 }
+                match filter.is_error {
+                    Some(is_error) if x.context.is_error != is_error => return false,
+                    None if x.context.is_error => return false,
```

Comment:
So by default we don't show the events on a operation with an execution that failed ? That's a different behavior then the others ones. If there is a good reason we can keep it but we need to put it on the help in the description of the CLI and in the API docs

Reply:
This was Damir's insight, thought process was that we don't want the current behaviour to change for users that are building stuff on massa. Current implementation is not breaking on input nor on output values

Reply:
I'm just saying that" not specifying a filter means "it filter" " : that is a different behavior than the other filters on the CLI so this needs to be specify.

As a side note you said "we don't want the current behaviour to change for users" but we are changing it here because the error message that is shown when an operation fail will now not be displayed when no filter. This is a change.

Reply:
Yeah true, tbh I don't have a strong opinion about what should be adopted here, both behaviours seem justified to me

Reply:
That's my opinion too